docs/contributing.rst

```diff
@@ -60,13 +60,28 @@ The HTML of the docs will be written to :file:`docs/_build/html`.
 Code style
 ^^^^^^^^^^
 
-Run ``tox -e format`` to automatically reformat your changes. Run
-``tox -e lint,types`` to see any remaining code smells or type errors
-that need to be fixed manually.
+To automatically reformat your changes with `isort`_ and `black`_, run:
 
-Additionally, the Twine maintainers prefer that ``import`` statements
-be used for packages and modules only, rather than individual classes
-or functions.
+.. code-block:: console
+
+  tox -e format
+
+To detect any remaining code smells with `flake8`_, run:
+
+.. code-block:: console
+
+  tox -e lint
```

Comment:
On other issues we've discussed indentation like this. The project currently uses 4 space indents and this is using 2.

Reply:
I recall that (#585), but the existing `code-block` directives were using 2, so I followed that pattern. I've updated them all in https://github.com/pypa/twine/pull/631/commits/de22c77a9fcbbd33a1ea3d96eb67c19c8e92170b.